src/test/unleash-proxy.test.ts

```diff
@@ -684,3 +684,65 @@ test('Should return 501 when all feature toggles is not enabled', () => {
         .set('Authorization', 'sdf')
         .expect(501);
 });
+
+describe('Request content-types', () => {
+    test.each(['/proxy', '/proxy/all'])(
+        'Should assign default content-type if the request has a body but no content-type (%s)',
+        async (endpoint) => {
+            const client = new MockClient();
+            const payload = {
+                context: { appName: 'my-app' },
+            };
+
+            const proxySecrets = ['sdf'];
+            const app = createApp(
+                {
+                    unleashUrl,
+                    unleashApiToken,
+                    proxySecrets,
+                    enableAllEndpoint: true,
+                },
+                client,
+            );
+            client.emit('ready');
+
+            await request(app)
+                .post(endpoint)
+                .set('Authorization', 'sdf')
+                .set('Content-Type', 'application/json')
```

Comment:
Is this well aligned with the test description? 
> Should assign default content-type if the request has a body but no content-type (%s)

According to that, I think we should not have this line

Reply:
Aha! You're perfectly right! That's a holdover from when I couldn't make it work. It's been addressed in 849084ebf735f74b2379007fe52aa9767f9f0ff2